src/components/NewExperiment.vue

```diff
@@ -1473,9 +1568,29 @@ function getInitialState() {
           measurement: strs => strs.map(str => parseFloat(str)),
           uncertainty: strs => strs.map(str => parseFloat(str))
         },
+        rules: {
+          sample: growthItem => [
+            this.requiredIfTableHasChanged(
+              this.tables.growth,
+              growthItem.sample
+            )
+          ],
+          measurement: growthItem => [
+            this.requiredIfTableHasChanged(
+              this.tables.growth,
+              growthItem.measurement
+            )
+          ]
+        },
         items: [{ temporaryId: uuidv4() }],
         isValid() {
-          return true;
+          const rules = this.rules;
+          const keys = Object.keys(rules);
+          return this.items.every(item =>
+            keys.every(key =>
+              rules[key](item).every(valid => typeof valid === "boolean")
+            )
+          );
```

Comment:
I guess try to reuse the same function, or remove isValid from tables and make methods: isTableValid(table)